experimental/ag-ui/server.py

```
@@ -0,0 +1,411 @@
+# Add CORS middleware front-end access
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+@app.get("/api/agui/chat/health")
+def agui_chat(request: Request):
```

Ruff (0.13.3)

Unused function argument: `request`

(ARG001)